test/fixtures/index.js

```diff
@@ -66,6 +66,7 @@ module.exports = {
   inputJpgLarge: getPath('giant-image.jpg'),
   inputJpg320x240: getPath('320x240.jpg'), // http://www.andrewault.net/2010/01/26/create-a-test-pattern-video-with-perl/
   inputJpgOverlayLayer2: getPath('alpha-layer-2-ink.jpg'),
+  inputJpgShrinkOnLoad: getPath('shrink-on-load-test.jpg'),
```

Comment:
What is the source of this test image? Are we granted the right to distribute it, e.g. via a suitable Creative Commons licence?